sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringModelFactory.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Azure.AI.Language.QuestionAnswering
 {
-    /// <summary> Model factory for read-only models. </summary>
+    /// <summary> Model factory for generated models. </summary>
```

Comment:
1. Why does it matter to the user that these are generated models? That's an implementation detail that should, in fact, be hidden from the user.
2. Might be more apt to say, "Model factory for response models with read-only properties."
3. We may hand-code these to augment them. That's why this is a partial class. Also saying these are "generated models" would be incorrect.

Reply:
I believe `Model factory for models with read-only properties.` should be better because we could also include input models.
On the other hand, since we could add customized code to this class, we might add some models without readonly properties, shall we use just `Model factory for models`?

Reply:
That works too, though with correct punctuation: "Model factory for models." The main point was to leave implementation details out of it since it doesn't really matter to the user.